Library/Formula/telegraf.rb

```diff
@@ -0,0 +1,141 @@
+require "formula"
+require "language/go"
+
+class Telegraf < Formula
+  desc "Server-level agent for InfluxDB"
+  homepage "https://influxdb.com"
+  url "https://github.com/influxdb/telegraf.git", :using => :git, :tag => "v0.1.1"
```

Comment:
`:using => :git` is redundant with a `.git` url. We should be using a tarball here unless there are submodules involved. If there are submodules, please use a `:tag` and a `:revision`.